autotest/gdrivers/mrf.py

```diff
@@ -362,6 +362,27 @@ def test_mrf_lerc_with_huffman():
     gdal.Unlink('/vsimem/out.lrc')
     gdal.Unlink('/vsimem/out.til')
 
+def test_raw_lerc():
+    if 'LERC' not in gdal.GetDriverByName('MRF').GetMetadataItem('DMD_CREATIONOPTIONLIST'):
```

Comment:
how is this test case different from https://github.com/OSGeo/gdal/blob/master/autotest/gdrivers/mrf.py#L47 ? Is OPTIONS:V1:1 different from OPTIONS=V1:YES ? If so, it could probably be added to the mrf_list array